pkg/cmd/kind/load/docker-image/docker-image.go

```diff
@@ -37,6 +38,10 @@ import (
 	"sigs.k8s.io/kind/pkg/internal/runtime"
 )
 
+var (
+	imageTagFetcher = nodeutils.ImageTags
```

Comment:
instead of a global, we can pass the function, or split the function under test into two and pass the results of calling this.

Reply:
Let me refactor is to pass the function across and use that instead. Would make it cleaner.